README.md

```diff
@@ -10,38 +10,42 @@ Utilities for [Hexo].
 
 ## Table of contents
 
-- [Installation](#installation)
-- [Usage](#usage)
-- [Cache](#cache)
-- [CacheStream](#cachestream)
-- [camelCaseKeys](#camelcasekeysobj-options)
-- [createSha1Hash](#createsha1hash)
-- [decodeURL](#decodeurlstr)
-- [deepMerge](#deepmergetarget-source)
-- [encodeURL](#encodeurlstr)
-- [escapeDiacritic](#escapediacriticstr)
-- [escapeHTML](#escapehtmlstr)
-- [escapeRegex](#escaperegexstr)
-- [full_url_for](#full_url_forpath)
-- [gravatar](#gravatarstr-options)
-- [hash](#hashstr)
-- [highlight](#highlightstr-options)
-- [htmlTag](#htmltagtag-attrs-text-escape)
-- [isExternalLink](#isexternallinkurl-sitehost-exclude)
-- [Pattern](#patternrule)
-- [Permalink](#permalinkrule-options)
-- [prettyUrls](#prettyurlsurl-options)
-- [prismHighlight](#prismhighlightstr-options)
-- [relative_url](#relative_urlfrom-to)
-- [slugize](#slugizestr-options)
-- [spawn](#spawncommand-args-options)
-- [stripHTML](#striphtmlstr)
-- [wordWrap](#wordwrapstr-options)
-- [tocObj](#tocobjstr-options)
-- [truncate](#truncatestr-options)
-- [unescapeHTML](#unescapehtmlstr)
-- [url_for](#url_forpath-option)
-- [bind(hexo)](#bindhexo)
+- [hexo-util](#hexo-util)
```

Comment:
It seems like you are using a VSCode extension that generates ToC automatically. Please disable it temporarily for the changes.

Reply:
I've solved this in b53f3fa